globals.js

```diff
@@ -0,0 +1,52 @@
+const { createLuxonDateTimeFromIso, createTimeZone } = require('./dateTimes');
+
+function getOrThrowIfMissingOrEmpty(configField) {
+    const value = process.env[configField];
+    if (!value) {
+        throw new Error(
+            `${configField} is required. Please create a .env file, based off of the .env.template file, and ensure that all variables have values (no empty quotes)`
+        );
+    }
+
+    return value;
+}
+
+function getDateTimesFromArgv(timeZone) {
+    console.info(`Using time zone: ${createTimeZone(timeZone).name}`);
+    const startDate = process.argv[2];
+    const endDate = process.argv[3];
+
+    const startMoment = createLuxonDateTimeFromIso(startDate, timeZone).startOf('day');
+    const endMoment = createLuxonDateTimeFromIso(endDate, timeZone).endOf('day');
+
+    return [startMoment, endMoment];
+}
+
+const githubToken = Buffer.from(getOrThrowIfMissingOrEmpty('GITHUB_TOKEN')).toString('base64');
```

Comment:
I don't (yet) understand why we are using Buffer and toString here.

Reply:
I don't remember, but I do remember double-checking that it made sense when the code was first written.
You could try looking at the PR where it was added and see if there are any helpful comments.

Reply:
It looks like it may have originally been necessary when we passed both an id and secret (instead of a token). Not sure though. It's fine for now. Maybe later we can simplify it.